WootricSDK/WootricSDK/Wootric.h

```diff
@@ -77,4 +78,9 @@
 
 + (void)setCustomValueForResurveyThrottle:(NSNumber *)resurveyThrottle visitorPercentage:(NSNumber *)visitorPercentage registeredPercentage:(NSNumber *)registeredPercentage dailyResponseCap:(NSNumber *)dailyResponseCap;
 
++ (void)setSendButtonBackgroundColor:(UIColor *)color;
++ (void)setNoThanksColor:(UIColor *)color;
++ (void)setSliderColor:(UIColor *)color;
++ (void)setSocialSharingColor:(UIColor *)color;
```

Comment:
Mmm rechecking this I realized that maybe it's better if I change this
`setSendButtonBackgroundColor:(UIColor *)color`
to this
`setSendButtonBackgroundColorWithRed:(CGFloat)red green:(CGFloat)green blue:(CGFloat)blue`
thus restricting users of doing things like setting alpha to 0 and making things invisible. On the other hand, it would prevent them from using other methods from the UIColor class, like `initWithCGColor:(CGColorRef)cgColor` and `initWithPatternImage:(UIImage*)image`, reducing their options when creating colors. @diablourbano @hachibu any thoughts on this?